hack/test_e2e.sh

```diff
@@ -41,24 +41,57 @@ GINKGO_PROVIDER="local"
 
 source "${SCRIPT_DIR}"/utils.sh
 source "${SCRIPT_DIR}"/common.sh
+source "${SCRIPT_DIR}"/test_skip_list.sh
+
+echo AAAAAA ${GINKGO_SKIP_ipv4_ebpf_TEST}
 
 function if_error_warning {
     ###########################################################################
     # Description:                                                            #
     # Validate if previous command failed and show an error msg (if provided) #
     #                                                                         #
     # Arguments:                                                              #
-    #   $1 - error message if not provided, it will just exit                 #
+    #   arg1 - error message if not provided, it will just exit               #
     ###########################################################################
     if [ "$?" != "0" ]; then
         if [ -n "$1" ]; then
-            RED="\e[31m"
+            RED="\e[91m"
             ENDCOLOR="\e[0m"
             echo -e "[ ${RED}FAILED${ENDCOLOR} ] ${1}"
         fi
     fi
 }
 
+function result_message {
+    ###########################################################################
+    # Description:                                                            #
+    # show [FAILED] in red and a message.                                     #
+    #                                                                         #
+    # Arguments:                                                              #
+    #   arg1 - result                                                         #
+    #   arg2 - message to output                                              #
+    ###########################################################################
+    [ $# -eq 2 ]
+    if_error_exit "result_message: $# Wrong number of arguments to ${FUNCNAME[0]}... args: $#"
+
+    local result=${1}
+    local message="${2}"
+
+    if [ -z "${result}" ]; then
+        echo "result_message() requires a message"
+        exit 1
+    fi
+    RED="\e[91m"
+    GREEN="\e[92m"    
+    ENDCOLOR="\e[0m"
+
+    if [ "${result}" == "0" ] ; then
+	echo -e "${GREEN}[SUCCES!] ${message} succeded!!! ${ENDCOLOR}"
```

Comment:
```suggestion
	echo -e "${GREEN}[SUCCESS!] ${message} succeded!!! ${ENDCOLOR}"
```

Reply:
I managed to fck something up, working on fixing it right now.